packages/adapters/wagmi/src/client.ts

```diff
@@ -470,11 +488,12 @@ export class WagmiAdapter implements ChainAdapter {
               'networkControllerClient:getApprovedCaipNetworksData - wagmiConfig is undefined'
             )
           }
-          const chainId = Number(
-            NetworkUtil.caipNetworkIdToNumber(this.appKit?.getCaipNetwork()?.id)
-          )
+          const activeCaipNetwork = this.appKit?.getCaipNetwork()
+          const chainId = activeCaipNetwork?.id as number | undefined
```

Comment:
These are duplicationg, maybe we could add `getCaipNetwork` function?

Reply:
`getCaipNetworkId` in any case I think